frontend/app/components/SessionIndividual/index.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React from 'react';
+import { cleanHtml } from 'utils';
+import type { SessionT } from 'types';
+import NotFound from 'NotFound';
+import withLogic from './logic';
+
+type SessionIndividualProps = {
+  session?: SessionT,
+};
+const SessionIndividual = (props: SessionIndividualProps) => {
+  if (!props.session) {
+    return <NotFound />;
```

Comment:
If this component is supposed to represent an single Session, I would expect this kind of logic to appear "up one" component, so that this Session component could be reused to display a session anywhere